.github/workflows/ci.yml

```diff
@@ -54,15 +54,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        setfile:
-          - name: Ubuntu 16.04
-            value: "ubuntu1604-64spipedserver.ma{hostname=spipedserver}-ubuntu1604-64spipedclient.a{hostname=spipedclient}"
-          - name: Ubuntu 18.04
-            value: "ubuntu1804-64spipedserver.ma{hostname=spipedserver}-ubuntu1804-64spipedclient.a{hostname=spipedclient}"
-          - name: Debian 8
-            value: "debian8-64spipedserver.ma{hostname=spipedserver}-debian8-64spipedclient.a{hostname=spipedclient}"
-          - name: Debian 9
-            value: "debian9-64spipedserver.ma{hostname=spipedserver}-debian9-64spipedclient.a{hostname=spipedclient}"
+        setfile: ${{fromJson(needs.setup_matrix.outputs.beaker_setfiles)}}
```

Comment:
Please revert these changes for now. This module uses multiple hosts and this was the easiest way to achieve them for now.